pkg/graph/resolvers/trb_advice_letter.go

```diff
@@ -77,15 +78,50 @@ func UpdateTRBAdviceLetter(ctx context.Context, store *storage.Store, input map[
 	return updatedLetter, err
 }
 
-// RequestReviewForTRBAdviceLetter sets a TRB advice letter as ready for review and (TODO) notifies the given recipients.
-func RequestReviewForTRBAdviceLetter(ctx context.Context, store *storage.Store, id uuid.UUID) (*models.TRBAdviceLetter, error) {
-	// TODO - EASI-2514 - send notification email(s)
-
+// RequestReviewForTRBAdviceLetter sets a TRB advice letter as ready for review and notifies the given recipients.
+func RequestReviewForTRBAdviceLetter(
+	ctx context.Context,
+	store *storage.Store,
+	emailClient *email.Client,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	id uuid.UUID,
+) (*models.TRBAdviceLetter, error) {
 	letter, err := store.UpdateTRBAdviceLetterStatus(ctx, id, models.TRBAdviceLetterStatusReadyForReview)
 	if err != nil {
 		return nil, err
 	}
 
+	trb, err := store.GetTRBRequestByID(ctx, letter.TRBRequestID)
+	if err != nil {
+		return nil, err
+	}
+
+	requesterInfo, err := fetchUserInfo(ctx, trb.GetCreatedBy())
+	if err != nil {
+		return nil, err
+	}
+
+	leadInfo, err := fetchUserInfo(ctx, *trb.TRBLead)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Will probably need to chuck this behind a `nil` check as well so we don't error out if there's no TRB lead